tests/puppeteer/pages/Install/install.js

```diff
@@ -133,6 +143,16 @@ module.exports = class INSTALL extends CommonPage {
    */
   async checkInstallationSuccessful() {
     await this.page.waitForSelector(this.installationProgressBar, {visible: true});
+    await this.page.waitFor(this.installationProgressBar, {visible: true});
+    await this.page.waitFor(this.generateSettingsFileStep, {visible: true, timeout: 90000});
+    await this.page.waitFor(this.installDatabaseStep, {visible: true, timeout: 90000});
+    await this.page.waitFor(this.installDefaultDataStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.populateDatabaseStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.configureShopStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installModulesStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installModulesAddons, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installThemeStep, {visible: true, timeout: 360000});
+    await this.page.waitFor(this.installFixturesStep, {visible: true, timeout: 360000});
```

Comment:
Please, don't make any changes of scripts in this PR.
And waiting for 6 min (or more) for installation is a lot.

Reply:
Do we really need all these timeouts, since we're checking every step ? Surely, none of the steps take more than 6 minutes to complete ?